### ISSUE TITLE
Suggestion: Toggles on compare view could indicate what they control

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1392]
PROBLEM:
For full screen desktop users, it's fairly clear what is controlled by each toggle group, however on mobile and especially as a screen reader user, it is confusing to see a bunch of unlabelled toggled groups side by side. 

SOLUTION:
For the demographic toggle, identify the second toggle; either `Demographic 2` or perhaps using the 2nd variable (either condition or FIPS)
![Screen Shot 2022-01-10 at 4 31 51 PM](https://user-images.githubusercontent.com/41567007/148855152-189c288c-31a6-4b80-b68e-50f3be0b2e4e.png)

Note: Implemented the meta-variable name in front of `DATA TYPE` for clarity in #1389  for both SR and brower users

**Comment by benhammondmusic** [comment]
This 2nd demo toggle should be under the 2nd population card 

![image](https://user-images.githubusercontent.com/41567007/149266570-d02fe87d-68e0-4598-a37f-e57b3415318a.jpeg)
